Gemfile

```diff
@@ -1,15 +1,10 @@
+# This gemfile provides additional gems for testing and releasing this cookbook
+# It is meant to be installed on top of ChefDK which provides the majority
```

Comment:
It's less than ideal sometimes, but it allows us to avoid constantly updating test deps. For instance the test deps included in this gem file are insanely old already and there's a good chance they're not going to run in modern ruby. Relying on ChefDK pushes that work to Chef and away from the cookbook maintainers.